app/models/repository_host/bitbucket.rb

```diff
@@ -201,5 +229,16 @@ def self.api_client(token = nil)
     def api_client(token = nil)
       self.class.api_client(token)
     end
+
+    def log_error(repo_name:, request:, error: nil)
+      log_info = {
+        external_service: "bitbucket",
+        request_label: request,
+        repository_full_name: repo_name,
+      }
+      log_info.merge!(error_message: error.message, error_name: error.name) if error
```

Comment:
If this method is for logging an error and it's only called when we're rescuing, is there a reason `error` would ever be `nil`?